src/main/java/rx/internal/schedulers/ImmediateScheduler.java

```diff
@@ -41,9 +42,6 @@ public Worker createWorker() {
 
         final BooleanSubscription innerSubscription = new BooleanSubscription();
 
-        InnerImmediateScheduler() {
```

Comment:
Yep! Making the whole class package scope will eliminate it as well. I saw
that you did this a few times in this PR as well.

On Thu, Jun 23, 2016 at 1:39 PM David Karnok <notifications@github.com>
wrote:

> In src/main/java/rx/internal/schedulers/ImmediateScheduler.java
> <https://github.com/ReactiveX/RxJava/pull/4091#discussion_r68280028>:
>
> > @@ -41,9 +42,6 @@ public Worker createWorker() {
> >
> >          final BooleanSubscription innerSubscription = new BooleanSubscription();
> >
> > -        InnerImmediateScheduler() {
>
> Would it help if I remove the private from the parent class?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/ReactiveX/RxJava/pull/4091/files/fa9b5d28e6ae895f794c16badc1740607a701fb6#r68280028>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAEEEc7OZhLEkjUWlD_XjKD1ixdmh-VKks5qOsTjgaJpZM4I8smQ>
> .
>